### ISSUE TITLE
Improve doc comment -> about translation when there are multiple lines and lists

### ISSUE THREAD

**Issue by fitzgen** [TeXitoi/structopt#69]
First off, thanks for writing structopt -- its awesome!

--------------------------------------

A doc comment like

```rust
/// Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin laoreet vel
/// velit vestibulum fringilla. Mauris rhoncus arcu eget justo pretium, vel
/// tempus ante mollis. Curabitur odio ligula, venenatis sed convallis sed,
/// viverra eu lectus.
///
/// * Mauris condimentum magna fringilla nulla lacinia vulputate. Integer
///   imperdiet finibus erat, eu molestie neque cursus sit amet. Mauris a
///   posuere enim.
///
/// * Mauris et felis elit. In ultrices convallis risus nec aliquam. In
///   convallis lorem imperdiet, dignissim arcu sit amet, sollicitudin lectus.
///
/// Aliquam nec dapibus nunc. Nunc semper molestie mauris sed aliquet. Duis
/// facilisis tristique varius. Ut ac lorem ut sem finibus aliquam. Nam vel
/// posuere ipsum, ac porttitor enim.
```

Results in a garbled `about` message.

I'd expect it to come out pretty much exactly as written, but maybe with the line wraps reflowed.

**Comment by TeXitoi** [comment]
Fixed in v0.2.3. That may not be as pretty as it can be in the bullet point, but such a feature would be a clap feature.

The only remaining downside is indenting that will be suppressed. Not sure that's relevant.

**Comment by fitzgen** [comment]
Wow! Fast turn around 😃
